sdk/appconfiguration/app-configuration/src/generated/src/appConfigurationContext.ts

```diff
@@ -6,14 +6,12 @@
  * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
```

Comment:
delete this "generated" folder and regenerate, and I shouldn't see context file.